### PR TITLE
Task-53938: Add file name when marking file as favorite

### PR DESCRIPTION
**Pull request by SaraBoutej**
In this PR, we add new extension to display favorite file in the top bar favorites drawer.

### CODE REVIEW

File: documents-webapp/src/main/webapp/vue-app/documents-extensions/components/DocumentsFavoriteItem.vue [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+    <v-list-item-content>
+      <v-list-item-title class="text-color body-2">{{ documentTitle }}</v-list-item-title>
+    </v-list-item-content>
+
+    <v-list-item-action>
+      <v-btn icon>
+        <v-icon class="yellow--text text--darken-2" size="18">fa-star</v-icon>
+      </v-btn>
+    </v-list-item-action>
```
Comment (AzmiTouil) [inline documents-webapp/src/main/webapp/vue-app/documents-extensions/components/DocumentsFavoriteItem.vue]: IMO, this should be added as a common button, out of register component. 
[https://github.com/Meeds-io/social/blob/develop/webapp/portlet/src/main/webapp/vue-apps/common/components/FavoriteButton.vue](url) component to use

Comment (AzmiTouil) [inline documents-webapp/src/main/webapp/vue-app/documents-extensions/components/DocumentsFavoriteItem.vue]: IMO, this should be added as a common button, out of register component. 
[https://github.com/Meeds-io/social/blob/develop/webapp/portlet/src/main/webapp/vue-apps/common/components/FavoriteButton.vue](url) component to use

**Review by AzmiTouil — CHANGES_REQUESTED** [review]

File: documents-webapp/src/main/webapp/WEB-INF/conf/documents/dynamic-container-configuration.xml [hunk 2]
```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright (C) 202 eXo Platform SAS.
```
Comment (AzmiTouil) [inline documents-webapp/src/main/webapp/WEB-INF/conf/documents/dynamic-container-configuration.xml]:
```suggestion
    Copyright (C) 2022 eXo Platform SAS.
```

**Review by AzmiTouil — CHANGES_REQUESTED** [review]

File: documents-webapp/src/main/webapp/vue-app/documents-extensions/components/DocumentsFavoriteItem.vue (hunk 1 above)
Comment (SaraBoutej) [reply documents-webapp/src/main/webapp/vue-app/documents-extensions/components/DocumentsFavoriteItem.vue]: done

Comment (SaraBoutej) [reply documents-webapp/src/main/webapp/vue-app/documents-extensions/components/DocumentsFavoriteItem.vue]: done

File: documents-webapp/src/main/webapp/WEB-INF/conf/documents/dynamic-container-configuration.xml (hunk 2 above)
Comment (SaraBoutej) [reply documents-webapp/src/main/webapp/WEB-INF/conf/documents/dynamic-container-configuration.xml]: done

**Pull request merged**